File: javascript/src/f.js
class F {

    constructor(team, f, a) {
        this.team = team;
        this.f = f;
        this.a = a;
    }

    r() {
        return this.team;
    }

    sort() {
        return Math.abs(this.f - this.a);
    }
}

module.exports = {
    F
};

File: java/src/main/java/algorithm/F.java
package algorithm;

public class F implements Data {

    public final String team;
    public final int f;
    public final int a;

    public F(String team, int f, int a) {
        this.team = team;
        this.f = f;
        this.a = a;
    }

    @Override
    public String r() {
        return team;
    }

    @Override
    public int sort() {
        return Math.abs(f - a);
    }
}
